docs/docs/build/contracts/index.md

```diff
@@ -0,0 +1,19 @@
+# Contracts
+
+## BOB Testnet (Sepolia)
+
+### Multicall3
+
+- Multicall3: `0x089b191d95417817389c8eD9075b51a38ca46DE8`
+
+### BOB Core Contracts
+
+- tBTC Relay: ``
+
+### Tokens (ERC20)
+
+- Erc20Minter (faucet): `0x7884560F14c62E0a83420F17832988cC1a775df1`
+- TBTC:
```

Comment:
Please deploy tBTC if we haven't done that yet

Reply:
Done